lms/envs/devstack.py

```diff
@@ -16,7 +16,8 @@
 SITE_NAME = 'localhost:8000'
 PLATFORM_NAME = ENV_TOKENS.get('PLATFORM_NAME', 'Devstack')
 # By default don't use a worker, execute tasks as if they were local functions
-CELERY_ALWAYS_EAGER = True
+CELERY_ALWAYS_EAGER = False
+FEATURES['ENABLE_SERVICE_STATUS'] = True
```

Comment:
This will be reverted, right?

Reply:
Yes, the first commit of this PR needs to be reverted